### DISCUSSION TITLE
Unarchived repo: Actions not running

### DISCUSSION THREAD

**Discussion by schoenwaldnils** [community/community#25405] [Actions]
<p>As said in the title, I unarchived a repo, pushed to master and no action is running.<br>
Any Idea how or why this is?</p>
<p>Repo in question: <a href="https://github.com/rueckenwind/cyclon" rel="nofollow noopener">https://github.com/rueckenwind/cyclon</a></p>
<p>Edit: a new commit did the trick :thinking:</p>
<pre><code>git commit --allow-empty -m "wakey wakey GitHub Actions"
</code></pre>
<p>Edit2: <br>
<a href="https://www.githubstatus.com/" rel="nofollow noopener">https://www.githubstatus.com/</a><br>
Seems the unarchiving is not related. There are some general issues right now. </p>

**Comment by hugobarzano** [comment]
<p>Same issue here, after push changes  to master no action is running… :( </p>

**Comment by pkaneshige** [comment]
<p>Seeing this issue on deploys to master as well as pull requests.</p>

**Comment by wrdaigle** [comment]
<p>My actions have been failing for the past hour or so.  Looks like github is working on it: <a href="https://www.githubstatus.com/" rel="nofollow noopener">https://www.githubstatus.com/</a></p>

**Comment by schoenwaldnils** [comment]
<p>Seems to be unrelated to my unarchived repo.</p>
<p><a href="https://www.githubstatus.com/" rel="nofollow noopener">https://www.githubstatus.com/</a> shows there are some issues right now.</p>

**Comment by BrightRan** [comment]
<p><a class="mention" href="/schoenwaldnils">@schoenwaldnils</a> <a class="mention" href="/hugobarzano">@hugobarzano</a> <a class="mention" href="/pkaneshige">@pkaneshige</a> <a class="mention" href="/wrdaigle">@wrdaigle</a>,<br>
Whether the problem has gone on your side?</p>

**Comment by schoenwaldnils** [comment, marked as answer]
<p>Had no problems again so far. Might have been the overall actions issues at that time.</p>
